src/app/home.component.ts

```diff
@@ -1,23 +0,0 @@
-import {
-  Component
-} from '@angular/core';
-
-import {
-  SkyAppStyleLoader
-} from './public';
-
-@Component({
-  selector: 'app-home',
-  template: 'asdf'
-})
-export class HomeComponent {
-  constructor(
-    private styleLoader: SkyAppStyleLoader
-  ) {
-    this.styleLoader.loadStyles().then(() => {
```

Comment:
We need to keep this style loader demo somewhere. You could put it in the **inputs-demo.component.ts** file.